CorsixTH/Lua/config_finder.lua

```diff
@@ -18,8 +18,10 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE. --]]
 
+dofile "system/path_manager"
```

Comment:
Should be require probably; we don't want to load this file twice.

Reply:
I got the same error with it not being local here as well